SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -70,17 +83,19 @@ public final class SwiftGenStoryboardEnumBuilder {
         }
     }
     
-    public func build(enumName enumName: String? = "Name", indentation indent : SwiftGenIndentation = .Spaces(4)) -> String {
+    public func build(scenesEnumName enumName: String? = "Scene", seguesEnumName: String? = "Segue", indentation indent : SwiftGenIndentation = .Spaces(4)) -> String {
```

Comment:
Now that we have two different `enums`, it doesn't even make sense anymore to let the option to the user to _not_ provide an `enumName` (which so far was possible and generated all the `enums` for the `Scenes` at the root of the `UIStoryboard` extension). So we better migrate those two parameters to non-optional `String`s